_data/integrations.yml

```diff
@@ -305,6 +295,7 @@
 
     - systran:
         custom: true
+        adaptive: true
```

Comment:
Since I added this, need to re-run generate.py

https://www.linkedin.com/posts/memoq-translation-technologies_as-of-memoq-100-neural-fuzzy-adaptation-activity-7077982362260713472-4oWg